### PR TITLE
Tag GMT.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [joa-quim/GMT.jl](https://github.com/joa-quim/GMT.jl)
Release: [v0.1.0](https://github.com/joa-quim/GMT.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/joa-quim/GMT.jl/builds/248774801)
Diff: [vs v0.0.4](https://github.com/joa-quim/GMT.jl/compare/c81699f9123c4fb299d5b76b61702f136fbb62a2...5250c46760d0916f114511a5a24a05d902240ca6)
`requires` vs v0.0.4: 
```diff
--- 0.0.4/requires
+++ 0.1.0/requires
@@ -1,2 +1,2 @@
-julia 0.4
-Compat 0.8.0
+julia 0.5
+Compat 0.26.0
```
cc: @joa-quim

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

File: GMT/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5
+Compat 0.26.0
```
Comment (tkelman) [inline GMT/versions/0.1.0/requires]: it doesn't look lile you're using anything newer than Compat.String, which you won't need if the package is 0.5-and-up

Comment (joa-quim) [inline GMT/versions/0.1.0/requires]: I just added the ``Compat`` version that I have installed with ``Pkg.update()``. Really, it's too much of a burden and time spending going look into ``Compat`` to see what's the minimum necessary version. If a ``Pkg.update()`` will install the necessary version, what's the problem of requiring a newer version?

Comment (tkelman) [inline GMT/versions/0.1.0/requires]: That prevents installation of working version combinations when versions are pinned or held back due to bounds. It takes a few minutes looking at blame. What features do you need? It doesn't look like you actually need Compat at all right now.

Comment (joa-quim) [inline GMT/versions/0.1.0/requires]: To get rid of deprecation warnings I remember that I had to at least replace calls of the type
``out = Array(Any, n_out)``  to ``out = Array{Any}( n_out)``

Comment (tkelman) [inline GMT/versions/0.1.0/requires]: that works on 0.5 without Compat

Comment (joa-quim) [inline GMT/versions/0.1.0/requires]: So what shall I do? Delete this PR, remove the Compat request and tag again?

Comment (tkelman) [inline GMT/versions/0.1.0/requires]: see the attobot readme for instructions, you can replace the tag and attobot will update this pr in place

**Comment by tkelman** [comment]
should also remove Compat from src and test if it's not in REQUIRE

**Pull request merged**